dataactcore/read_config.py

```diff
@@ -0,0 +1,165 @@
+import os
+import os.path
+from os.path import expanduser, normpath, dirname, abspath
+import yaml
+import re
+
+VALUES_TO_LOG = {"CONFIG_BROKER":["path","use_aws","local",'aws_bucket', 'aws_role', 'aws_region', 'aws_create_temp_credentials',
```

Comment:
Sure, I can do that.  Only difficulty is if we want to allows things to be marked for logging in multiple files.  As long as we can maintain the fields_to_log dict in one config this is a simple change, but if we want to combine different dicts from multiple files that would be inconsistent with the dict methods proposed below.  Would the assumption here be that if you want to log a field from the environment specific config, you'll either add it to the base config, or overwrite the list in the base config by relisting all fields to be logged in the environment-specific config?  Either of those is consistent with the overwriting approach we've been taking so far, I just want to make sure we're not expecting a different behavior for this field.